tests/test_enterprise/api/test_throttles.py

```diff
@@ -49,6 +49,8 @@ def test_user_throttle(self):
         """
         Make sure throttling works as expected for regular users.
         """
+        self.create_user('test_user', 'QWERTY')
+        self.client.login(username='test_user', password='QWERTY')
```

Comment:
needed to change normal client `user` so that other tests do not get affected by consumption of throttle limit.
it was **not** failing before because we used to have different client IDs for every test class.